### PR TITLE
About Page general cleanup and improvements.

### PR DESCRIPTION
**Pull request by weerd**
### What's this PR do?

This pull request adds a new `coverImageMediaQueryStyles()` helper function to help DRY up the page cover image media query styles (thanks @lindsayrmaher  for the suggestion!).

I also renamed the `message` prop for the `BannerCallToAction` to just `text` which is more of the naming pattern we use in other components (having a `title` and a `text` prop). It also moves more of the copy content into the JSON content file.

### How should this be reviewed?

👀 

### Any background context you want to provide?

I had some weird rebasing issue, but while it looks like a lot of changes it technically wasn't that many 😅 

### Relevant tickets

References [Pivotal #]().

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: resources/assets/components/pages/AboutPage/AboutPage.js [hunk 1]
```diff
@@ -3,240 +3,196 @@ import { Fragment, React } from 'react';
...
+
+          <div
+            className="bg-gray-400"
+            css={css`
+              background-position: center center;
+              background-repeat: no-repeat;
+              background-size: cover;
+              height: 540px;
+
+              ${coverImageMediaQueryStyles(data.header.coverImage.url)}
```
Comment (weerd) [inline resources/assets/components/pages/AboutPage/AboutPage.js:37]: @lindsayrmaher new helper function you can also use for the Articles page. It returns a string of the styles, so just dump it into an emotion `css` string:
```
`css={css`...`}
```

File: resources/assets/components/pages/AboutPage/AboutPage.js [hunk 2]
```diff
@@ -3,240 +3,196 @@ import { Fragment, React } from 'react';
...
+                text="Join Now"
+              />
+            </div>
+          </div>
 
-            <div className="col-span-4 md:col-span-8 lg:col-span-10 xl:col-span-6 lg:col-start-2 xl:col-start-2 xl:row-start-1 mt-4 xl:mt-0">
-              <h2 className="mb-0 text-lg text-purple-400 uppercase">
-                About Us
-              </h2>
+          <AnalyticsWaypoint name="RENAME_section_bottom" />
```
Comment (weerd) [inline resources/assets/components/pages/AboutPage/AboutPage.js:67]: Upcoming analytics PR will handle this renaming 😄

File: resources/assets/components/utilities/CallToAction/BannerCallToAction.js [hunk 3]
```diff
@@ -9,8 +9,8 @@ const BannerCallToAction = ({
   attributes,
   children,
   colorClasses,
-  message,
   stacked,
+  text,
```
Comment (weerd) [inline resources/assets/components/utilities/CallToAction/BannerCallToAction.js:13]: Renamed this so it's more in line with other component's props names with similar behavior.

File: resources/assets/components/pages/AboutPage/AboutPage.js (hunk 1 above)
Comment (lindsayrmaher) [reply resources/assets/components/pages/AboutPage/AboutPage.js:37]: love it!

File: resources/assets/components/pages/AboutPage/AboutPage.js [hunk 4]
```diff
@@ -3,240 +3,196 @@ import { Fragment, React } from 'react';
...
+          data-test="newsletter-section"
+        >
+          <StrikeThroughHeader title="Get Inspired. Get Entertained. Get Active." />
+
+          <div className="grid-wide text-center">
+            <NewsletterSubscriptionForm />
+          </div>
+        </section>
+
+        {/* Member Highlight Section */}
```
Comment (lindsayrmaher) [inline resources/assets/components/pages/AboutPage/AboutPage.js:136]: appreciate these labels!

File: resources/assets/components/pages/HomePage/HomePage.js [hunk 5]
```diff
@@ -151,35 +152,6 @@ const HomePageTemplate = ({
...
-            '539',
-            'fill',
-          )});
-        }
-      `
-    : null;
-
   return (
     <Fragment>
       {/* @TODO: Once EmotionJS supports shorthand syntax for React.Fragment, switch <Fragment> out for <> syntax! */}
```
Comment (lindsayrmaher) [inline resources/assets/components/pages/HomePage/HomePage.js:157]: interesting that EmotionJS doesn't support the shorthand, i didn't realize that 👀

**Review by lindsayrmaher — APPROVED** [review]
Not seeing anything specific I'd change, just some observations!

File: resources/assets/components/pages/HomePage/HomePage.js (hunk 5 above)
Comment (weerd) [reply resources/assets/components/pages/HomePage/HomePage.js:157]: Ya, I tried using the shorthand again on the About page to see if maybe an update patch of EmotionJS might have fixed it, but no dice 😢

**Pull request merged**